lib/.conan-recipes/llnl-units/conanfile.py

```diff
@@ -39,7 +38,7 @@ class UnitsConan(ConanFile):
     def source(self):
         git = tools.Git("units")
         git.clone("https://github.com/SimonHeybrock/units.git")
-        git.checkout('9fc2d62')
+        git.checkout('c3dab73')  # this is LLNL/units@v0.6.0 with a namespace fix
```

Comment:
Not super happy about this. But given the long time between releases, I am fine with it.

Reply:
I had opened an upstream PR this morning, will see when it is included.